docs/aws/nRFCloudLocationServices.rst

```diff
@@ -80,3 +80,26 @@ Use the following commands:
     npx cdk deploy --all
 
 This will update the StateMachine, which resolves P-GPS request from devices to use the nRF Cloud API as a resolver.
+
+Ground Fix Services
+*******************
+
+You can enable your devices to request the nearest Wi-Fi access points using `nRF Cloud's Ground Fix Service API <https://api.nrfcloud.com/v1#tag/Ground-Fix>`_ in your deployment.
```

Comment:
This change does not enable the devices. It enables a feature in nRF Asset Tracker.

```suggestion
You can enable `nRF Cloud's Ground Fix Service API <https://api.nrfcloud.com/v1#tag/Ground-Fix>`_ to geolocate Wi-Fi site surveys sent by devices in your deployment.
```